### ISSUE TITLE
Error 404 after running 'npm link overcooked'

### ISSUE THREAD

**Issue by jxu81** [HumanCompatibleAI/overcooked-hAI-exp#8]
Hello,

I got the following errors when I tried to link overcooked js package. I hope someone can point out some directions. Thanks

npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/overcooked - Not found
npm ERR! 404 
npm ERR! 404  'overcooked@*' is not in this registry.
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

**Comment by micahcarroll** [comment]
Did you install overcooked-js from [this version](https://github.com/HumanCompatibleAI/overcooked_ai/tree/6eaceb0a9a2501f1b9fccbf4c7016d6662ed1108) of the overcooked_ai repo?

**Comment by jxu81** [comment]
No, I will do that now. Also, may I know the version of js when this code was used/tested last time?

**Comment by micahcarroll** [comment]
I'm not entirely sure (I'm guessing it might be documented somewhere in the repo linked above) – but (to my knowledge) it was probably last tested in late 2020.
